volontulo/urls.py

```diff
@@ -41,6 +41,11 @@
         name='offer_form'
     ),
     url(
+        r'^offers/form/(?P<organization_id>[0-9]+)/(?P<offer_id>[0-9]+)$',
```

Comment:
If I have no offer_id in URL there is offers/form/organization_id URL used instead.